src/test/java/com/adobe/target/edge/client/service/TelemetryServiceTest.java

```diff
@@ -302,6 +306,52 @@ void testTelemetryForServerSideSendNotification() throws NoSuchFieldException {
     assertEquals(1, telemetryServiceSpy.getTelemetry().getEntries().size());
   }
 
+  /**
+   * Test case for calling sendNotifications() and getting an http timeout. In this case Delivery
+   * API will return a null response body, and the SDK will throw a RuntimeException, and not
+   * collect telemetry.
+   *
+   * @throws NoSuchFieldException
+   */
+  @Test
+  void testTelemetryForServerSideSendNotificationNullResponse() throws NoSuchFieldException {
+    setup(
+        true,
+        DecisioningMethod.SERVER_SIDE,
+        "testTelemetryForServerSideSendNotificationNullResponse");
+    Mockito.lenient()
+        .doReturn(getNoContentDeliveryResponse())
+        .when(defaultTargetHttpClient)
+        .execute(any(Map.class), any(String.class), any(DeliveryRequest.class), any(Class.class));
+    Context context = getContext();
+
+    List<Notification> notifications = new ArrayList<>();
+    Notification notification =
+        new Notification()
+            .id("12345")
+            .impressionId("12345")
+            .mbox(new NotificationMbox().name("test-mbox").state("11111"))
+            .type(MetricType.DISPLAY)
+            .timestamp(System.currentTimeMillis());
+    notifications.add(notification);
+
+    TargetDeliveryRequest targetDeliveryRequest =
+        TargetDeliveryRequest.builder().context(context).notifications(notifications).build();
+
+    try {
+      targetJavaClient.sendNotifications(targetDeliveryRequest);
```

Comment:
Let's actually check if the exception is thrown here, otherwise we never know for sure, see https://stackoverflow.com/a/41386406 for example